#
# Description: Update libupnp API for 1.6.x.
#   Between libupnp2 and libupnp3 (pupnp 1.4.x and 1.6.x) the function
#   SetLogFileNames was renamed to UpnpSetLogFileNames.
# Author: nick@leverton.org
# Bug-Debian: 581879 
#
--- djmount/fuse_main.c	2010-05-16 21:40:56.000000000 +0100
+++ djmount/fuse_main.c	2010-05-16 21:43:55.000000000 +0100
@@ -616,7 +616,7 @@
 	}  
 	Log_Colorize (true);
 #if UPNP_HAVE_DEBUG
-	SetLogFileNames ("/dev/null", "/dev/null");
+	UpnpSetLogFileNames ("/dev/null", "/dev/null");
 #endif
 	
 	/*
@@ -710,10 +710,10 @@
 					Log_SetMaxLevel (LOG_ERROR);
 #if UPNP_HAVE_DEBUG
 				} else if (strcmp (s, "upnperr") == 0) {
-					SetLogFileNames ("/dev/stdout", 
+					UpnpSetLogFileNames ("/dev/stdout",
 							 "/dev/null");
 				} else if (strcmp (s, "upnpall") == 0) {
-					SetLogFileNames ("/dev/stdout", 
+					UpnpSetLogFileNames ("/dev/stdout",
 							 "/dev/stdout");
 #endif
 				} else {
